tensorflow_lite_support/python/task/vision/core/utils.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utils for Vision APIs."""
+
+from tensorflow_lite_support.python.task.core.proto import configuration_pb2
+
+
+def build_proto_base_options(proto_options, options):
+  """Builds initial proto options values from the base options."""
+
+  proto_options.model_file_with_metadata.file_name = options.base_options.model_file
+  proto_options.num_threads = options.base_options.num_threads
```

Comment:
I'm not sure I understand why you're doing this (filling deprecated options from their newer counterpart). Cc @khanhlvg in case I'm missing something.

Reply:
@mbrenon Uh the image embedder python module seemed to be using the same so I figured I'd use that. I'll look into it and make sure it uses the new counterpart.

Reply:
Got it. Indeed, the image embedder options don't use `BaseOptions` at all so this logic is required there, but it's not required here.

Reply:
I've fixed it and made sure it uses the newer BaseOptions and ComputeSettings counterparts.